server/src/grpc/server.rs

```diff
@@ -337,4 +418,26 @@ mod tests {
         let result: ReadyResponse = result.unwrap().into_inner();
         assert_eq!(result.ready, true);
```

Comment:
Came across this one a couple of times myself. My local clippy notes it should be simplified like this:
`assert!(result.ready);`

Reply:
568a2a762168b9f1fe611860989da50cc02c494a